subjects/quad/README.md

```diff
@@ -1,4 +1,4 @@
-# quadrangle
+## quadrangle
```

Comment:
why did you change this ? Is it for display reason on the platform ?

Structurally it made more sense for it to be H1, maybe we should lower the following titles to H3 instead of H2

Reply:
I changed it to be the same as the other projects, for consistency, but if the structure is not relevant in this case we can let it be an H1 just to distinguish the main tittle better

Reply:
![quad-readme](https://user-images.githubusercontent.com/53977751/160869426-bec055f3-d9ac-4421-8079-60997869eb69.png)

This is what it will look like